components/home/sections/OCUsers.js

```diff
@@ -112,7 +112,7 @@ const User = ({ id, name, picture, type, collectivePath }) => {
             {intl.formatMessage(messages[`home.OCusers.${id}`])}
           </P>
         </Box>
-        <HomeStandardLink width="72px" href="/discover">
```

Comment:
At least according to the document linked in the issue, it's not what we want.

Reply:
Yes. I've removed the static '/discover' and replaced it with the collectivePath prop

Reply:
According to the Copy document linked in the issue this is not what we want.

https://docs.google.com/document/d/1i1eP5PLfM2Tshp6Oiqnpt8lXAazJ-twFj8iQQg2SYWI

Reply:
Updated the links!

Reply:
Thank you so much. Looks good! Gonna test quickly.

Reply:
np :)